Nodejs/Product/Nodejs/SharedProject/CommonPropertyPage.cs

```diff
@@ -347,9 +32,9 @@ void IPropertyPage.SetObjects(uint count, object[] punk)
             {
                 if (punk[0] is ProjectConfig)
                 {
-                    if (this._project == null)
+                    if (this.Project == null)
                     {
-                        this._project = (CommonProjectNode)((CommonProjectConfig)punk.First()).ProjectMgr;
+                        this.Project = (CommonProjectNode)((CommonProjectConfig)punk.First()).ProjectMgr;
```

Comment:
Why did we test for `ProjectConfig` but cast to `CommonProjectConfig`?

Reply:
bad copy/paste